internal/services/dataprotection/data_protection_backup_policy_postgresql_resource_test.go

```diff
@@ -22,6 +23,26 @@ type DataProtectionBackupPolicyPostgreSQLResource struct{}
 func TestAccDataProtectionBackupPolicyPostgreSQL_basic(t *testing.T) {
 	data := acceptance.BuildTestData(t, "azurerm_data_protection_backup_policy_postgresql", "test")
 	r := DataProtectionBackupPolicyPostgreSQLResource{}
+	if !features.FivePointOhBeta() {
```

Comment:
These tests need to be split into 4.x and 5.0 versions, since they have differing functionality, and currently they will require significant effort to resolve/make right when 5.0 ships.

Reply:
Fixed.